examples/jsm/loaders/3MFLoader.js

```diff
@@ -47,9 +47,16 @@ class ThreeMFLoader extends Loader {
 		super( manager );
 
 		this.availableExtensions = [];
+		this.defaultMaterialColor = 0xaaaaff;
```

Comment:
Can we define this as a `THREE.Color` object? `setDefaultMaterialColor()` would be defined like so then:

> this.defaultMaterialColor.set( defaultMaterialColor );

It's more consistent if color properties are actually of type `THREE.Color` like with materials.

Reply:
Good point, I've modified it to store a `THREE.Color` object.